docs/reference/resource-limits-fees.mdx

```diff
@@ -19,11 +19,7 @@ Note that write fees grow linearly from empty ledger to ledger "target size", an
 
 The ledger rent cost ('Write 1KB' entries in the table) is based on the write fee, rent period and some coefficient. For the temporary storage, the coefficient is 10 months (in ledgers), thus 1 month of temporary entry rent is 1/10 of the rent fee. For persistent storage the coefficient is 1 month (in ledgers) and thus 1 month of rent is equivalent to the write fee.
 
-### Phase 2 {#phase-2}
-
-Phase 2 marks the deployment of a user-ready Mainnet where anyone can deploy and interact with smart contracts. In this phase, the network is ready for more extensive smart contract utilization, so it's the practical beginning of smart contracts on the Stellar network. But even this is not the end of the journey. At the core of smart contracts on the Stellar network is scale and sustainability, and efforts to continue to expand capacity and introduce further capabilities to enhance the network's functionalities will be ongoing. See [this blog post](https://stellar.org/blog/developers/the-stellar-network-s-phased-rollout-of-smart-contracts-the-road-to-mainnet) for more details and share your feedback in [Discord](https://discord.com/invite/zVYdY3ktTn).
-
-| Network setting | Phase 2 Cost (stroops) |
+| Network setting | Cost (stroops) |
```

Comment:
Hey @dmkozh thanks for the feedback.

Does this commit address the following correctly?:
> I don't think we need to separate this by phase, let's just update the values in-place.